javav2/example_code/eventbridge/src/main/java/com/example/eventbridge/HelloEventBridge.java

```diff
@@ -1,66 +1,65 @@
-//snippet-sourcedescription:[ListEventBuses.java demonstrates how to list your Amazon EventBridge buses.]
-//snippet-keyword:[AWS SDK for Java v2]
-//snippet-service:[Amazon EventBridge]
-
-/*
-   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-   SPDX-License-Identifier: Apache-2.0
-*/
-
-package com.example.eventbridge;
-
-// snippet-start:[eventbridge.java2._list_buses.import]
-import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider;
-import software.amazon.awssdk.regions.Region;
-import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
-import software.amazon.awssdk.services.eventbridge.model.EventBridgeException;
-import software.amazon.awssdk.services.eventbridge.model.EventBus;
-import software.amazon.awssdk.services.eventbridge.model.ListEventBusesRequest;
-import software.amazon.awssdk.services.eventbridge.model.ListEventBusesResponse;
-import java.util.List;
-// snippet-end:[eventbridge.java2._list_buses.import]
-
-/**
- * Before running this Java V2 code example, set up your development environment, including your credentials.
- *
- * For more information, see the following documentation topic:
- *
- * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
- */
-public class ListEventBuses {
-
-    public static void main(String[] args) {
-
-        Region region = Region.US_WEST_2;
-        EventBridgeClient eventBrClient = EventBridgeClient.builder()
-            .region(region)
-            .credentialsProvider(ProfileCredentialsProvider.create())
-            .build();
-
-        listBuses(eventBrClient) ;
-        eventBrClient.close();
-    }
-
-    // snippet-start:[eventbridge.java2._list_buses.main]
-    public static void listBuses( EventBridgeClient eventBrClient) {
-
-        try {
-            ListEventBusesRequest busesRequest = ListEventBusesRequest.builder()
-                .limit(10)
-                .build();
-
-            ListEventBusesResponse response = eventBrClient.listEventBuses(busesRequest);
-            List<EventBus> buses = response.eventBuses();
-            for (EventBus bus: buses) {
-                System.out.println("The name of the event bus is: "+bus.name());
-                System.out.println("The ARN of the event bus is: "+bus.arn());
-            }
-
-        } catch (EventBridgeException e) {
-
-            System.err.println(e.awsErrorDetails().errorMessage());
-            System.exit(1);
-        }
-    }
-    // snippet-end:[eventbridge.java2._list_buses.main]
-}
+//snippet-sourcedescription:[HelloEventBridge.java demonstrates how to list your Amazon EventBridge buses.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-service:[Amazon EventBridge]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.eventbridge;
+
+// snippet-start:[eventbridge.java2._list_buses.import]
```

Comment:
Why does this file have such a weird snippet pattern? `java2._list_buses` has a `._`

Reply:
Some of these are inherited. I typically do not use _ chars. They work - so I leave them